src/components/DesignDemo/index.tsx

```
@@ -1,57 +1,116 @@
+  className?: string;
+}) {
+  return (
+    <div className="flex flex-col items-center gap-1">
+      <div
+        className={cn(
+          'w-20 h-20 rounded-lg border border-(--color-border)',
+          className,
+        )}
+        style={{ backgroundColor: `var(${color})` }}
+      />
+      <span className="text-[11px] text-(--color-ink-3) font-mono">{label}</span>
```

ESLint

[error] Replace `{label}` with `⏎········{label}⏎······`

(prettier/prettier)